README.md

```diff
@@ -0,0 +1,75 @@
+# OCA Addons Repo Template
+
+This is a template created to make easier the task of maintaining OCA addon
+repositories.
+
+## Why?
+
+We have dozens of repos. Most of them look the same, and most of them need
+specific-but-similar configurations for CI, code quality, dependency management, etc.
+
+We need a place where to evolve those things and push them automatically everywhere
+else.
+
+This is that place.
+
+## How to use?
+
+This is a template. It is based on [Copier](https://github.com/pykong/copier), go there
+to read its docs to know how it works.
+
+Quick answer to bootstrap a new repo:
+
+```bash
+# Install copier and pre-commit if missing
+pipx install copier
+pipx install pre-commit
+pipx ensurepath
+# Clone this template and answer its questions
+copier copy https://github.com/OCA/oca-addons-repo-template.git some-repo
+# Commit that
+cd some-repo
+git add .
+pre-commit install
+pre-commit run -a
+git commit -am 'Hello world 🖖'
+```
+
+Quick answer to update a repo:
+
+```bash
+# Update the repo
+cd some-repo
+copier update
+# Reformat updated files
+pre-commit run
+# Commit update
+git commit -am 'Updated from template'
+# Reformat all other files, in case some pre-commit configuration was updated
+pre-commit run -a || git commit -am 'Reformatted after template update'
+```
+
+## How to contribute?
+
+Go read [our contribution guideline](CONTRIBUTING.md).
+
+## Supported use cases
+
+This template allows to bootstrap and update addon repositories for these Odoo versions:
+
+- 13.0
+
+Future versions will be added as they are released. Past versions could be added as long
+as they don't break existing branches.
+
+Right now this template is tightly coupled with code, guidelines and decisions from OCA.
+You might find some things that you can reuse in your own templates, but in general
+terms this template is not meant to support being used as is for other organizations.
+
+## The legal stuff
+
+Copyright holder: [Odoo Community Association](https://odoo-community.org/).
+
+Template license: [MIT](LICENSE)
+
+License of the rendered repositories: [AGPL](LICENSE.jinja)
```

Comment:
The repos themselve don't really have a license, since each addon can have a different one.

Reply:
But they do have... example: https://github.com/OCA/server-tools/blob/12.0/LICENSE

Reply:
Yeah, but that's confusing. Look for instance in the `queue_job` repo which has a mix of LGPL and AGPL addons, but the repo file says AGPL.

Reply:
Yes, I don't quite understand that... Might be the licensing for files outside of a module? We can take this chance to state clearly in the README that each module has its own license. :shrug:

Reply:
I'm thinking... Is it actually legal to combine modules licensed with, say, Apache-2.0, in a repo that is itself licensed under AGPL-3?

Maybe we should license our repositories under MIT or something like that? :thinking: 

Are we allowed to do such change at this point of history? Or better for v14+?

Reply:
These days I'm using https://choosealicense.com/licenses/bsl-1.0/ which is "Apache but shorter", could be a good option too.

Reply:
I think the current situation is just because most people don't pay attention to that.
I'd say it's totally allowed to mix licences in one repo, as long as the license for each file is clear.
I'm not sure it's important to have a specific license for these dotfiles at the repo root.

Reply:
Well, AFAIK you're not allowed to mix any \*GPL license with code that doesn't respect the fundamental rights they enforce. At least not the other thing depending on the \*GPL thing, which would be our case. Except for LGPL.

For <= Odoo v8 this wouldn't matter because the base was AGPL, so anything built on it had to be AGPL too.

For Odoo v9+ it can be a problem. But changing this at this point will lead to an endless discussion with all the community.

I think we should just start using another license for v14+ repos. Let's leave things as they are now.